crates/eframe/src/web/events.rs

```diff
@@ -77,11 +77,16 @@ pub(crate) fn install_event_handlers(runner_ref: &WebRunner) -> Result<(), JsVal
     // so we check if we have focus inside of the handler.
     install_copy_cut_paste(runner_ref, &document)?;
 
-    install_mousedown(runner_ref, &canvas)?;
     // Use `document` here to notice if the user releases a drag outside of the canvas:
     // See https://github.com/emilk/egui/issues/3157
     install_mousemove(runner_ref, &document)?;
-    install_mouseup(runner_ref, &document)?;
+    if is_mobile() {
+        install_pointerup(runner_ref, &canvas)?;
+        install_pointerdown(runner_ref, &canvas)?;
+    } else {
+        install_mouseup(runner_ref, &document)?;
+        install_mousedown(runner_ref, &canvas)?;
+    }
```

Comment:
Please add a comment here explaining why we do different things on mobile and web. In particular, why doesn't `pointerup/down` work on desktop?

Reply:
I did this because I'm not sure how to properly test it and make sure it works properly for other things as well, but from my naive testing, it looks like it works just fine

Reply:
If you change the `install_pointerup` to target `document`, does it still work?

Reply:
I already changed it locally, yes it works, will push it in a bit

Reply:
Okay, in that case I think we can remove the whole branch altogether, and leave only `install_pointer*`

Reply:
ok, review latest commit please (I force pushed)